_posts/2015-10-19-twis-38.md

```diff
@@ -0,0 +1,29 @@
+---
+layout:     post
+title:      This Week In Servo 38
+date:       2015-10-19 20:30:00
+summary:    What's up with Servo for the week of 13 Oct 2015 - 19 Oct 2015
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2015-10-12..2015-10-19+user%3Aservo),
+we landed 82 PRs in the Servo organization's repositories. Additionally, we passed 5,000 stars on https://github.com/servo/servo!
+
+###Notable additions
+
+ - jdm added a [description of our directories and most important dependencies](https://github.com/servo/servo/pull/8054), with help from ms2ger
+ - frewsxcv moved some of our [old ref tests](https://github.com/servo/servo/pull/8045) to the new WPT harness
+ - frewsxcv also [improved caching](https://github.com/servo/servo/pull/8036) on Servo Travis builds
+
```

Comment:
https://github.com/servo/servo/pull/8020 and https://github.com/servo/servo/pull/8060 (Made heap `JS<T>` pointers dereferencable)